recommender/api/endpoints/health_check.py

```diff
@@ -0,0 +1,16 @@
+"""Health check endpoint definition"""
+
+from __future__ import annotations
+from flask_restx import Resource, Namespace
+from recommender.services.health_check import deep_health_check
+
+api = Namespace("health", "Endpoint used for checking the application's health")
+
+
+@api.route("")
+class HealthCheck(Resource):
+    """Groups methods for checking the application health"""
+
+    def get(self):
+        """Perform a deep health_check"""
```

Comment:
`Perform a deep health_check` is in the swagger endpoint description, Add here also the value of `HEALTHY_SERVICES_THRESHOLD, HEALTHY_USERS_THRESHOLD, K` with a description of what are those (someone else will be monitoring our service) -> of course only if the value will be set by us (not dynamically passed in the request body)